client/layout/masterbar/style.scss

```diff
@@ -73,6 +73,7 @@ $autobar-height: 20px;
 	border: solid 2px var( --color-masterbar-background );
 	border-radius: 50%;
 	display: none;
+	// stylelint-disable-next-line scales/font-size
```

Comment:
Not changed. Had to disable because the linter wouldn't let me commit otherwise.